test/integration/AaveV2.t.sol

```diff
@@ -52,7 +52,7 @@ contract AaveV2IntegrationTest is Test {
     uint256 public constant SIGNER_REFERRAL = 1;
     IAaveV2Provider public constant AAVE_V2_PROVIDER = IAaveV2Provider(0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5);
     IERC20 public constant USDC = IERC20(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48);
-    address public constant AUSDC_V2 = 0xBcca60bB61934080951369a648Fb03DF4F96263C;
+    address public constant AUSDC_V2_POOL = 0xBcca60bB61934080951369a648Fb03DF4F96263C;
```

Comment:
I don't quite get what is the purpose of this naming change. Since that this address is indeed an AToken and pool is a term that is used in lending pool already, turns out making it more confusing.

Reply:
This is in the Asana ticket. Both ways of writing for me are fine, what is Jeff's idea?

Reply:
Yes i did look into the ticket, however still don't feel that it improves the readability.

Reply:
Oh! I was thinking something like this

```
flashloanPool = AUSDC_V3;
amountIn = bound(amountIn, 1, token.balanceOf(flashloanPool));
```
For the original version, readers need to know people deposit USDC in AUSDC_V3 and AAVE uses AUSDC_V3 as a flashloan pool first to understand why we use `token.balanceOf(AUSDC_V3)` as the upperbound here.

Reply:
Oh, that's what it means, I think it's ok, it's been updated.